docs/general/community_packages.md

```diff
@@ -0,0 +1,39 @@
+![SeAT](http://i.imgur.com/aPPOxSK.png)
+
+# Community Packages
+
+Below is a list of packages contributed by the community. These packages normally follow the same installation procedure, however, its best you consult the documentation of the package itself in case there is anything special you need to get it working.
+
+## Generic package installation
+Packages will normally come in the form of a composer package that you need to include in your SeAT install, as well as a Service Provider that you need to bootstrap. So, generically speaking, installing a package will mean that you:
+
+- *Require* the package via composer:
+
+```bash
+composer require <package vendor>/<package-name>
+```
+
+- Bootstrap the packages service provider [here](https://github.com/eveseat/seat/blob/master/config/app.php#L184).
+
+Installing packages like this will ensure that none of the core SeAT packages are affected and you should be free to upgrade SeAT core at anytime.
+
+## Package list
+
+### SeAT specific packages
```

Comment:
Generally i would create 3 tables based on compatibility. Since SeAT 2.0 won't be much of a help for anyone i suggest these three:

1. Guaranteed compatibility with SeAT 3.0
2. Possible compatibility with SeAT 3.0
3. Not compatibel with SeAT 3.0

Reply:
don't really care about prior 3.0 package in fact since previous version are non longer able to work.
will short the thing with "mainteinaed" and "deprecated"

Reply:
`maintained`